README.md

```diff
@@ -12,6 +12,10 @@ HTTP2 to HTTP is supported too.
 npm i fastify-reply-from
 ```
 
+## Compatibility with fastify-multipart
+`fastify-reply-from` and [`fastify-multipart`](https://github.com/fastify/fastify-multipart) should not be registered as sibling plugins nor shold be registered in plugins which have a parent-child relationship.<br> The two plugins are incompatible, in the sense that the behavior of `fastify-reply-from` might not be the expected one when the above-mentioned conditions are not respected.<br>
+However, the two plugins may be used within the same fastify instance, at the condition that they belong to disjoint branches of the fastify plugins hierarchy tree. 
```

Comment:
It would be good to explain why they are incompatible, i.e. the original multipart request is consumed when parsed.